packages/manager/modules/netapp/src/dashboard/volumes/dashboard/template.html

```diff
@@ -0,0 +1,274 @@
+                >
+                    <oui-tile-description data-ng-if="!$ctrl.isEditing.name">
+                        <span data-ng-bind="$ctrl.volume.name"></span>
+                        <oui-button
+                            size="s"
+                            variant="link"
+                            on-click="$ctrl.onEditNameClick()"
+                        >
+                            <span
+                                class="sr-only"
+                                data-translate="netapp_volumes_dashboard_information_update"
+                            ></span>
+                            <span
+                                class="oui-icon oui-icon-pen"
+                                aria-hidden="true"
+                            ></span>
+                        </oui-button>
+                    </oui-tile-description>
+                    <oui-tile-description
+                        data-ng-if="$ctrl.isEditing.name"
+                        class="oui-input-group oui-input-group_inline"
+                    >
+                        <label
+                            for="volumeName"
+                            data-translate="netapp_volumes_dashboard_information_name"
+                            class="sr-only"
+                        ></label>
+                        <input
+                            class="oui-input"
+                            id="volumeName"
+                            name="volumeName"
+                            data-ng-model="$ctrl.volume.name"
+                        />
+                        <oui-button
+                            size="s"
+                            on-click="$ctrl.update($ctrl.volume)"
+                        >
+                            <span
+                                class="sr-only"
+                                data-translate="netapp_volumes_dashboard_information_confirm"
+                            ></span>
+                            <span
+                                class="oui-icon oui-icon-ok"
+                                aria-hidden="true"
+                            ></span>
+                        </oui-button>
+                        <oui-button size="s" on-click="$ctrl.onEditNameClick()">
+                            <span
+                                class="sr-only"
+                                data-translate="netapp_volumes_dashboard_information_cancel"
+                            ></span>
+                            <span
+                                class="oui-icon oui-icon-close"
+                                aria-hidden="true"
+                            ></span>
+                        </oui-button>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <oui-tile-definition
+                    term="{{:: 'netapp_volumes_dashboard_information_description' | translate }}"
```

Comment:
The logic and UI is similar to that of edit name. Can we create a re-usable component here?